src/ctlib/unittests/rpc_ct_setparam.c

```diff
@@ -414,14 +415,14 @@ int i, j;
 				coldata[i].value[0] = 0;
 				if (coldata[i].value == NULL) {
 					fprintf(stderr, "malloc coldata.value failed \n");
-					return 1;
+					break;
```

Comment:
I think you need also to set ret to `CS_FAIL`, otherwise you won't report an error

Reply:
I'll take another look, thanks.  Same goes for `lang_ct_param.c`, I suppose?

Reply:
> I'll take another look, thanks. Same goes for `lang_ct_param.c`, I suppose?

where exactly?

Reply:
Never mind, I misremembered which pair of tests wound up having parallel fixes.